[2025-05-07T02:16:42.131+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lgbtq_model_pipeline.retrain manual__2025-05-07T02:14:21.244768+00:00 [queued]>
[2025-05-07T02:16:42.136+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lgbtq_model_pipeline.retrain manual__2025-05-07T02:14:21.244768+00:00 [queued]>
[2025-05-07T02:16:42.136+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-05-07T02:16:42.144+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): retrain> on 2025-05-07 02:14:21.244768+00:00
[2025-05-07T02:16:42.149+0000] {standard_task_runner.py:57} INFO - Started process 284 to run task
[2025-05-07T02:16:42.152+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'lgbtq_model_pipeline', 'retrain', 'manual__2025-05-07T02:14:21.244768+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/lgbtq_model_pipeline.py', '--cfg-path', '/tmp/tmpteizfhy5']
[2025-05-07T02:16:42.154+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask retrain
[2025-05-07T02:16:42.180+0000] {task_command.py:416} INFO - Running <TaskInstance: lgbtq_model_pipeline.retrain manual__2025-05-07T02:14:21.244768+00:00 [running]> on host 6fdcfcafebfc
[2025-05-07T02:16:42.217+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='lgbtq_model_pipeline' AIRFLOW_CTX_TASK_ID='retrain' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T02:14:21.244768+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-07T02:14:21.244768+00:00'
[2025-05-07T02:16:42.218+0000] {logging_mixin.py:154} INFO - üì¶ Combining data...
[2025-05-07T02:16:42.421+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_246
[2025-05-07T02:16:42.422+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_247
[2025-05-07T02:16:42.422+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_245
[2025-05-07T02:16:42.423+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_249
[2025-05-07T02:16:42.423+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_248
[2025-05-07T02:16:42.424+0000] {logging_mixin.py:154} INFO - ‚ö†Ô∏è Skipping duplicate article_244
[2025-05-07T02:16:42.424+0000] {logging_mixin.py:154} INFO - üìö Loading labeled data...
[2025-05-07T02:16:42.541+0000] {logging_mixin.py:154} INFO - üß† Preparing triplet training data...
[2025-05-07T02:16:42.552+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/tasks/retrain.py", line 53, in train_triplet_model
    triplet_dataset = get_triplet_dataset(texts, labels, model_path=HF_MODEL_PATH)
  File "/opt/airflow/dags/model_code/model.py", line 64, in get_triplet_dataset
    tokenizer = AutoTokenizer.from_pretrained(model_path)
  File "/home/airflow/.local/lib/python3.8/site-packages/transformers/models/auto/tokenization_auto.py", line 939, in from_pretrained
    return tokenizer_class_fast.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/transformers/tokenization_utils_base.py", line 2197, in from_pretrained
    raise EnvironmentError(
OSError: Can't load tokenizer for '/opt/***/hf_model'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure '/opt/***/hf_model' is the correct path to a directory containing all relevant files for a GPT2TokenizerFast tokenizer.
[2025-05-07T02:16:42.558+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=lgbtq_model_pipeline, task_id=retrain, execution_date=20250507T021421, start_date=20250507T021642, end_date=20250507T021642
[2025-05-07T02:16:42.564+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 17 for task retrain (Can't load tokenizer for '/opt/***/hf_model'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure '/opt/***/hf_model' is the correct path to a directory containing all relevant files for a GPT2TokenizerFast tokenizer.; 284)
[2025-05-07T02:16:42.581+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-05-07T02:16:42.594+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
